INSTRUCTIONS.md

```diff
@@ -246,15 +265,15 @@ Now is **your turn** to come up with inventive ways of using neurophysiological
 - Series of introductory lectures on Brain-Computer Interfacing: [http://sccn.ucsd.edu/wiki/Introduction_To_Modern_Brain-Computer_Interface_Design](http://sccn.ucsd.edu/wiki/Introduction_To_Modern_Brain-Computer_Interface_Design)
 - A visualizer in [Unity](https://github.com/syswsi/BCI_Experiments)
 - [Using spotify and the Muse](https://github.com/eisendrachen00/musemusic)
-- Other Neurotech [ressource](https://github.com/NeuroTechX/awesome-bci)
-
-
+- Other NeurotechX [resource](https://github.com/NeuroTechX/awesome-bci)
 
 ## Authors
 
 Hubert Banville & Raymundo Cassani
 
-Thanks to the [MuSAE Lab](http://musaelab.ca/), [District 3](http://d3center.ca/) and [BCI Montréal](http://bcimontreal.org/). Thanks also to Ana, Sydney, Rohit and William for initial feedback on the workshop. 
+Thanks to the [MuSAE Lab](http://musaelab.ca/), [District 3](http://d3center.ca/) and [BCI Montréal](http://bcimontreal.org/). Thanks also to Ana, Sydney, Rohit and William for initial feedback on the workshop.
```

Comment:
Let's change the link to NeuroTechMTL's? :P
'[NeuroTechMTL](http://mtl.neurotechx.com/)'